HappyFox Test Automation Task Java /HelpDeskProductPageObject/src/main/java/Admin_Portal/Login_Page.java

```diff
@@ -0,0 +1,72 @@
+package Admin_Portal;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+import webdriverbase.AppPage;
+
+static class BasePage {
+        protected WebDriver driver;
+
+        public BasePage(WebDriver driver) {
+            this.driver = driver;
+        }
+    }
+
+    static class LoginPage extends BasePage {
+        public LoginPage(WebDriver driver) {
+            super(driver);
+        }
+
+        public void login(String username, String password) {
+
+            driver.findElement(By.id("username")).sendKeys(username);

```

Comment:
Elements are not declared it should be declared and make use out through out the project